content/people/collaboration.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "People"
+date: 2021-06-28T13:44:37-05:00
+draft: false
+layout: page
+aliases: /team/
+---
+We proudly present the geographic diversity of the TARDIS Research Group, a testament to our global collaboration and intercultural synergy. The map below illustrates the locations of our highly dedicated and skilled team members, scattered across the globe. Each pinpoint on the map represents not just an individual, but a unique perspective, a wealth of cultural wisdom, and a set of innovative ideas that contribute to the extraordinary dynamism and success of TARDIS. This rich tapestry of global backgrounds has helped TARDIS to break barriers, foster creativity, and generate groundbreaking research that crosses traditional geographical boundaries.
```

Comment:
This paragraph is a lot of corporate nonsense and refers to a map that doesn't exist. I recommend something simpler such as

"The TARDIS collaboration works together to develop and use TARDIS for scientific applications. Collaboration members have a range of cultural backgrounds, and we believe this makes TARDIS a stronger collaboration."